lib/sauce-browser.js

```diff
@@ -140,7 +140,7 @@ SauceBrowser.prototype.start = function () {
         self.emit('start', reporter)
 
         var timeout = false
-        var get_timeout = setTimeout(function () {
+        var timer = setTimeout(function () {
```

Comment:
Nice. This was bugging me :)

Reply:
Hehe same :) I really wanted to use `timeout` for the handle.